app/data_mapper_setup.rb

```diff
@@ -0,0 +1,12 @@
+require 'data_mapper'
+require 'dm-validations'
+require './app/models/user'
+require './app/models/peep'
+
+env = ENV['RACK_ENV'] || 'development'
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_challenge_#{env}")
```

Comment:
Metrics/LineLength: Line is too long. [98/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)